util/src/overlaydb.rs

```diff
@@ -14,19 +14,26 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Disk-backed `HashDB` implementation.
-
-use error::*;
-use hash::*;
-use bytes::*;
-use rlp::*;
-use hashdb::*;
-use memorydb::*;
-use std::ops::*;
-use std::sync::*;
-use std::env;
-use std::collections::HashMap;
+//! Disk-backed, `HashDB` implementation.
```

Comment:
Should be made crystal clear that this DB only supports unique values at the point of commit; i.e. at the point of commiting, each value may be in the backing database either 0 or 1 times.